go-controller/pkg/cni/helper_linux.go

```diff
@@ -666,6 +666,7 @@ func (*defaultPodRequestInterfaceOps) UnconfigureInterface(pr *PodRequest, ifInf
 	}
 
 	ifnameSuffix := ""
+	isSecondary := pr.netName != types.DefaultNetworkName
```

Comment:
will this only be true for ovn-k when we are hot plugging out a link interface? 
From my testing, we always have the default interface (eth0) and that interface/network is the first interface for CNI Del to tear down. We remove all the interfaces for that CNI DEL

Reply:
thats true we remove the eth0 and the primary UDN interface, but we dont remove secondary networks that are added using a network attachment. Those come in a different CNI request.